### ISSUE TITLE
Wpływ zmiany kategorii w produkcie na oferte

### ISSUE THREAD

**Issue by dlis2010** [allegro/allegro-api#3430]
Witam,

zmieniając cenę w aukcji otrzymaliśmy:
`Kategoria oferty jest różna od kategorii produktu - 258690.`

Rozumie, że zmieniła się kategoria do której był przypisany produkt podpięty pod ofertę.
Jaki powinien być prawidłowy "flow" w takich sytuacjach?

Powinniśmy całkowicie osunąć taką ofertę i wystawić ponownie w nowej kategorii?
Czy w powyższej sytuacji - gdy dochodzi do zmiany kategorii w przypisanym produkcie nie ma żadnej innej możliwości jak ponowna aukcja bo rozumie, że kategorii w ofercie nie zmienię (powyżej 24h)?

**Comment by PrzemyslawLukanowski** [comment]
Tak, na tę chwilę to jedyne rozwiązanie. Zaadresowałem ten problem do odpowiedniego zespołu.
